index.bs

```diff
@@ -244,10 +244,6 @@ if <var>request</var>'s <a for=request>header list</a>
    policy-controlled feature</a>, returns <code>false</code>, then skip the next steps and
    continue to the next <var>hintName</var>.
    [[!PERMISSIONS-POLICY]] [[!CLIENT-HINTS]]
-
-  <li><p>Set <var>hintName</var> to "Sec-" concatenated with <var>hintName</var>.
-  <div class=issue>We need to figure out if we really want a `Sec-` prefix, and if so also exempt it from CORS.</div>
```

Comment:
Should we file an issue related to the CORS question, assuming it's still an open issue?

Reply:
There's an open [PR](https://github.com/whatwg/fetch/pull/1000) on Fetch.